index.js

```diff
@@ -106,10 +106,14 @@ function readTag(buffer, offset, bigEndian) {
       return null;
     }
   }
-  
+
   // Special case for ascii strings
   if (type === 2) {
-    var string = buffer.toString('ascii', valueOffset, valueOffset + numValues);
+    var asciiSlice = buffer.slice(valueOffset, valueOffset + numValues);
+    if (asciiSlice.some(x => x >> 7 > 0))
+      return { buffer: asciiSlice, bigEndian: bigEndian };
```

Comment:
Given `bigEndian` applies to the EXIF data as a whole rather than each property, perhaps we should promote it as a top level property, which would then allow us to return the `Buffer` directly here.
```diff
- return { buffer: asciiSlice, bigEndian: bigEndian };
+ return asciiSlice;
```
```
{
  bigEndian: true
  image: {
    ImageDescription: <Buffer ...>
  }
}
```

Reply:
Makes sense, see 3039c9f.